opm/simulators/wells/MultisegmentWellSegments.cpp

```diff
@@ -735,13 +736,18 @@ pressureDropAutoICD(const int seg,
     }
 
     using MathTool = MathToolbox<EvalWell>;
-    const EvalWell mixture_viscosity = MathTool::pow(water_fraction, aicd.waterViscExponent()) * water_viscosity
-                                     + MathTool::pow(oil_fraction, aicd.oilViscExponent()) * oil_viscosity
-                                     + MathTool::pow(gas_fraction, aicd.gasViscExponent()) * gas_viscosity;
+    // make sure we don't pass negative base to powers
+    auto safe_pow = [](const auto a, const double b) {
```

Comment:
Tiny nit: This definition will copy `a` into the lambda.  That's never a problem if `a` turns out to be a scalar type like `double` but **could** become a concern if the underlying type is `EvalWell` or similar.  If you want to, could probably write this as
```C++
auto safe_pow = [](const auto& a, const double b) { ... };
```
with a "reference-to-`const`" parameter instead.  On the other hand I don't know that it matters too much.

Reply:
Thanks, done.